api/models.py

```diff
@@ -247,6 +247,13 @@ class Node(DatabaseModel):
     holdoff: Optional[datetime] = Field(
         description="Node expiry timestamp while in Available state"
     )
+    user: Optional[str] = Field(
+        description="Username of node owner"
+    )
+    user_groups: List[str] = Field(
+        default=[],
+        description="User groups that are permitted to update node"
+    )
```

Comment:
Oh I thought we wanted to just have one group, not a list of groups.  I guess this is fine but a bit more complicated.

Reply:
How is this related?  I like to think of Unix file systems, users can belong to a number of groups but files have only one group.

The current implementation seems to be working so that's fine, we can always simplify things by reducing it to a single group later on.